### PR TITLE
Show new stats at the character level

### PR DESCRIPTION
**Pull request by bhollis**
image

Feels pretty busy. I'd love to redesign this so we don't necessarily display these all the time.

### CODE REVIEW

**Comment by inexorableAce** [comment]
I'll take a look at this when I get home, I may actually just do a mock-up of the entire character emblem / dropdown as a whole. Lot of new stuff there.

File: src/app/inventory/store/stats.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ const statsNoBar = [
 ];
 
 /** Stats that all armor should have. */
-const armorStats = [
+export const armorStats = [
```
Comment (duckmanBR) [inline src/app/inventory/store/stats.ts]: Am I missing anything or the "new" stats should be listed here?

Comment (bhollis) [reply src/app/inventory/store/stats.ts]: That change was in a different branch

**Comment by bhollis** [comment]
It feels like we need a character popup...

**Comment by inexorableAce** [comment]
That was actually going to be what I mocked up tbh, a sheet with character details. Would make showing loadouts easier / nicer too as we could include some more details on them / give them their own tab etc

**Comment by inexorableAce** [comment]
ok quick mock-up based on the other sheet mock-ups we have and other PRs that have simplified the character emblem. Emblem stays clean just the bare essentials, tapping the emblem opens a character based sheet with character buffs, debuffs, stats, actions, and loadouts, since loadouts could have it's own tab like this we could show those in more depth like whats in them and such on that card. 

chars

**Comment by bhollis** [comment]
Other bits of info that might be of interest include the Title (wayfarer, etc) and the emblem counter if there is one. In game the emblem counter is part of your external customization.

**Comment by bhollis** [comment]
Also while you're in there, the tile needs some new affordance for clickability. I was thinking of making an arrow next to it, instead of on the tile where it gets lost. Lots of people don't realize it clicks.

**Comment by inexorableAce** [comment]
Agreed, I haven’t figured that part out yet but I have a mock-up of arrow next to that I’ll post when I get home in a sec. even that seems like it might not work? I dunno, also once it’s a card a down arrow seems... not right?

**Comment by inexorableAce** [comment]
Maybe vertical dots? Feels like a lot. 
image

here is without the white bar
image

**Comment by duckmanBR** [comment]
Not on mobile, but on desktop those 3 dots will look like a "grab here to custom reorder characters"...

**Comment by inexorableAce** [comment]
Yeah I think for mobile we have a different plan. Still dunno about desktop though.

**Comment by duckmanBR** [comment]
Can't we just make the chevron smaller and put it on the same line as the store name?  Since it's not a real button there's no need to be big.

**Comment by inexorableAce** [comment]
Doesn’t matter where we put it or what we use. Basically just need something to convey that this thing is clickable / tappable and it does things. There are a lot of people that don’t even know the emblems have interaction.

**Comment by duckmanBR** [comment]
Make it blinky until the user clicks it then never again... :rofl: 
But when we implement the DIM Database, something similar would be nice (teach the user that the the emblem have a menu).

**Comment by kyleshay** [comment]
I like the idea of moving it out and claiming back that space. Less on the main screen the better, especially for newer users.

Originally, those numbers in Destiny 1 had a tooltip that actually told you the exact number of seconds that it would take for your melee/super/grenade to cooldown. They were still somewhat useful to see at a glance in D2, but not as much. Now that we have more stats it may be more useful to make the usefulness of that into the loadout optimizer and instead placing those stats in a sheet like ace has recommended.

Whenever the stats translate into something meaningful, we'll definitely have to highlight that knowledge in DIM!

**Comment by bhollis** [comment]
Updated this to merge with the new 2-line stats. I'll wait until those stats actually start being returned.

**Comment by kyleshay** [comment]
Had another thought, if we want to keep the numbers there, we could make those stats/numbers not be sticky so you only see the main character header as you scroll.

**Comment by bhollis** [comment]
That's a neat idea - pin them to the top of the inventory column rather than to the card!

**Pull request merged**